Userland/Libraries/LibVT/CharacterSet.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, The SerenityOS Developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace VT {
+
+enum CharacterSet {
+    Iso_8859_1,
+    Null,
+    UserDefined,
+    VT100,
+};
+
+class CharacterSetTranslator {
+public:
+    u32 translate_code_point(CharacterSet active_set, u32 code_point)
```

Comment:
Maybe `translate_vt100_alternate_character_codepoint_to_unicode()`?

Reply:
Not really - it should also support three other sets apart from VT100, so I could extract a method for just VT100 but I'd rather see a nicer implementation of translation tables.